matplotlib-tests/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: matplotlib-tests
+  version: 1.5.0
+
+source:
+  git_url: https://github.com/matplotlib/matplotlib.git
+  git_rev: v1.5.x
+
+requirements:
+  build:
+    - matplotlib ==1.5.0
```

Comment:
Might be something like this: https://github.com/conda/conda-build/pull/8

I haven't used it, and don't see it used much, but I like the idea.